avalon/pipeline.py

```diff
@@ -1313,6 +1335,67 @@ def get_repres_contexts(representation_ids, dbcon=None):
     return contexts
 
 
+def get_subset_contexts(subset_ids, dbcon=None):
+    """Return parenthood context for subset.
+
+    Args:
+        subset_ids (list): The subset ids.
+        dbcon (AvalonMongoDB): Mongo connection object. `avalon.io` used when
+            not entered.
+
+    Returns:
+        dict: The full representation context by representation id.
+
+    """
+    if not dbcon:
+        dbcon = io
+
+    contexts = {}
+    if not subset_ids:
+        return contexts
+
+    _subset_ids = []
+    for id in subset_ids:
+        if isinstance(id, six.string_types):
+            id = io.ObjectId(id)
+        _subset_ids.append(id)
+
+    subset_docs = dbcon.find({
+        "type": "subset",
+        "_id": {"$in": list(subset_ids)}
```

Comment:
Misused variable `"subset_ids"` should be `"_subset_ids"`.

Reply:
Good catch! Fixed.